src/components/app/create/CreateApp.tsx

```diff
@@ -226,17 +228,92 @@ export class AddNewApp extends Component<AddNewAppProps, AddNewAppState> {
         });
     };
 
+    noOptionsMessage(inputObj: { inputValue: string }): string {
+        if (inputObj && (inputObj.inputValue === '' || inputObj.inputValue.length < 3)) {
+            return 'Type 3 chars to see matching results';
+        }
+        return 'No matching results';
+    }
+
+    appListOptions = (inputValue: string): Promise<[]> =>
+        new Promise((resolve) => {
+            if (this.timeoutId) {
+                clearTimeout(this.timeoutId);
+            }
+            this.timeoutId = setTimeout(() => {
+                if (inputValue.length < 3) {
+                    resolve([]);
+                    return;
+                }
+                getAppList({
+                    appNameSearch: inputValue,
+                    sortBy: 'appNameSort',
+                    sortOrder: 'ASC',
+                    size: 50,
+                })
+                    .then((response) => {
+                        let appList = [];
+                        if (response.result && !!response.result.appContainers) {
```

Comment:
Putting `response.result.appContainers` in if statement will check for undefined & null, no need of `!!`. Also, make sure to put a check around arr length if needed.

Reply:
`Array.isArray` can also be used in that matter to see if it's an array.